warehouse/bigquery/bigquery.go

```diff
@@ -251,6 +251,49 @@ func (bq *HandleT) dropStagingTable(stagingTableName string) {
 	}
 }
 
+//Need to create a structure with delete parameters instead of simply adding a long list of params
+func (bq *HandleT) DeleteBy(tableNames []string, params warehouseutils.DeleteByParams) error {
+	pkgLogger.Infof("BQ: Cleaning up the followng tables in bigquery for BQ:%s : %v", tableNames)
+	for _, tb := range tableNames {
+		if tb == "rudder_discards" {
+			continue
+		}
+		sqlStatement := fmt.Sprintf(`DELETE FROM "%[1]s"."%[2]s" WHERE 
```

Comment:
Format + Parameterization.

Reply:
I have given the parameters for the values and string for the names. Its the usual standard approach isn't it?

Reply:
This is done.